diambra/arena/make_env.py

```diff
@@ -17,8 +18,9 @@ def make(game_id, env_settings={}, wrappers_settings={},
     logging.basicConfig(level=log_level)
     logger = logging.getLogger(__name__)
 
-    # Include game_id in env_settings
+    # Include game_id and render_mode in env_settings
     env_settings["game_id"] = game_id
```

Comment:
If we make env_settings a EnvironmentSettings class, we could assign `env_settings.game_id = ..`, right? That would be nicer

Reply:
I definitely see your point, this is strictly related to the point above. Resolving this as it will be solved by the previous one.